INSTALL.md

```diff
@@ -165,3 +165,21 @@ To prune the number of cached images run `make prune`. There are two options tha
 
 - `PRUNE_HOURS` - the number of hours from when an image was last used (a cache hit) for it to be considered for pruning. The default is 48 hrs.
 - `PRUNE_KEEP` - the minimum number of cached images to keep in the cache. Default is 24 images.
+
+## CI workflow and options
+Every PR and every merge to master triggers the CI process in Jenkins(http://jenkins.rook.io).
```

Comment:
a few doc suggestions:
- Add links Jenkins and the s3 bucket in that link below
- wording in the next line: "... run integration tests, and publish artifacts on every push to a PR and master."
- "no artifacts are published" (artifacts is plural)